{"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cipherTextSize"},{"text":"(","kind":"text"},{"text":"value","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:Si","kind":"typeIdentifier","text":"Int"},{"text":")","kind":"text"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/SwiftKyber\/documentation\/SwiftKyber","doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException"]]},"sections":[],"identifier":{"url":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/cipherTextSize(value:)","interfaceLanguage":"swift"},"metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"cipherTextSize","kind":"identifier"},{"text":"(","kind":"text"},{"text":"value","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Int","kind":"typeIdentifier","preciseIdentifier":"s:Si"},{"text":")","kind":"text"}],"externalID":"s:10SwiftKyber0B9ExceptionO14cipherTextSizeyACSi_tcACmF","role":"symbol","modules":[{"name":"SwiftKyber"}],"title":"KyberException.cipherTextSize(value:)","roleHeading":"Case","symbolKind":"case"},"variants":[{"paths":["\/documentation\/swiftkyber\/kyberexception\/ciphertextsize(value:)"],"traits":[{"interfaceLanguage":"swift"}]}],"abstract":[{"type":"text","text":"Wrong cipher text size"}],"schemaVersion":{"patch":0,"minor":3,"major":0},"references":{"doc://SwiftKyber/documentation/SwiftKyber/KyberException/cipherTextSize(value:)":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"cipherTextSize","kind":"identifier"},{"text":"(","kind":"text"},{"text":"value","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Int","preciseIdentifier":"s:Si","kind":"typeIdentifier"},{"text":")","kind":"text"}],"identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/cipherTextSize(value:)","abstract":[{"type":"text","text":"Wrong cipher text size"}],"url":"\/documentation\/swiftkyber\/kyberexception\/ciphertextsize(value:)","role":"symbol","kind":"symbol","title":"KyberException.cipherTextSize(value:)","type":"topic"},"doc://SwiftKyber/documentation/SwiftKyber":{"abstract":[{"type":"text","text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard"}],"type":"topic","title":"SwiftKyber","role":"collection","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber","url":"\/documentation\/swiftkyber","kind":"symbol"},"doc://SwiftKyber/documentation/SwiftKyber/KyberException":{"role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"KyberException"}],"identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException","kind":"symbol","navigatorTitle":[{"text":"KyberException","kind":"identifier"}],"abstract":[{"type":"text","text":"The Kyber exceptions"}],"type":"topic","url":"\/documentation\/swiftkyber\/kyberexception","title":"KyberException"}}}